### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by phikal**
Here are some suggestions related to #21.  Feel free to change anything if you don't like it, the patch is intentionally opinionated.

### CODE REVIEW

**Comment by AndersonTorres** [comment]
@phikal I have included only the first file. The second throws syntax errors I am trying to figure out.

**Comment by phikal** [comment]
Interestingly it did not complain on my end (Emacs 30), but I have reinspected the commit and changed a few things.

File: bqn-symbols-doc.el [hunk 1]
```diff
@@ -3025,23 +2921,22 @@ at 9 ↕ \"short\"
...
      0 0 0 0 0 0
      1 0 1 0 1 0
-                 ┘"]
+                 ┘"]))
 
-      ;; ================================================
-      ;; Group
-"⊔"
+;;;;; Group
+                   (("⊔" .
```
Comment (AndersonTorres) [inline bqn-symbols-doc.el:2927]: @phikal why are you using two opening parens here, when in the other similar lines like 3044 you use only one?

**Comment by AndersonTorres** [comment]
P.S.: I took some liberty to use your patches. I will try to add authorship info as soon as possible.

**Comment by AndersonTorres** [comment]
I will close this, given that I "rebased" it onto the other PR.

**Pull request closed without merging**